sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java

```diff
@@ -60,9 +61,10 @@ public void useAzureKeyCredentialAsyncClient() {
      * Code snippet for getting async client using AAD authentication.
      */
     public void useAadAsyncClient() {
+        DefaultAzureCredential defaultCredential = new DefaultAzureCredentialBuilder().build();
```

Comment:
```suggestion
        TokenCredential tokenCredential = new DefaultAzureCredentialBuilder().build();
```

Reply:
I saw this ref page using DefaultAzureCredential. https://github.com/Azure/azure-sdk-for-java/wiki/Identity-and-Authentication
 
Should I follow it or use TokenCredential?

Reply:
It's better to use `TokenCredential` rather than a specific implementation. The builder says that `credential()` requires TokenCredential.